CMakeLists.txt

```diff
@@ -377,7 +377,10 @@ if(WITH_PROMETHEUS)
 endif()
 
 if(WITH_ABSEIL)
-  find_package(absl CONFIG REQUIRED)
+  if(TARGET absl::strings)
```

Comment:
nit: put a NOT to negate the condition in `if()` so the empty if block can be avoided?

Reply:
I have modified it.